manifests/repo/community.pp

```diff
@@ -0,0 +1,50 @@
+# @summary Private class to manage sensu community repository resources
+# @api private
+#
+class sensu::repo::community {
+
+  case $facts['os']['family'] {
+    'RedHat': {
+      if $facts['os']['name'] == 'Amazon' {
+        if $facts['os']['release']['major'] =~ /^201\d$/ {
+          $repo_release = '6'
+        } else {
+          $repo_release = '7'
+        }
+      } else {
+        $repo_release = $facts['os']['release']['major']
+      }
+      yumrepo { 'sensu_community':
+        ensure          => 'present',
+        baseurl         => "https://packagecloud.io/sensu/community/el/${repo_release}/\$basearch",
+        descr           => 'sensu_community',
+        enabled         => 1,
+        gpgcheck        => 0,
+        gpgkey          => 'https://packagecloud.io/sensu/community/gpgkey',
+        metadata_expire => 300,
+        repo_gpgcheck   => 1,
+        sslcacert       => '/etc/pki/tls/certs/ca-bundle.crt',
+        sslverify       => 1,
+}
```

Comment:
style, i think this paran is misaligned..

Reply:
It's not, at least doesn't look like it in VIM.  Everything lines up with `metadata_expire` which is longest parameter name.

Reply:
check you aren't using hard tabs..  this paran is definitely off as it should line up with `yumrepo`

Reply:
I fixed that when switching to if statements.